recipes/repeatmasker/RepeatMaskerConfig.pm

```diff
@@ -0,0 +1,183 @@
+    $REPEATMASKER_LIB_DIR $WUBLAST_DIR $WUBLASTN_PRGM
+    $WUBLASTP_PRGM $WUBLASTX_PRGM $SETDB_PRGM $XDFORMAT_PRGM 
+    $DEMAKE $DECYPHER $LIBPATH $TRF_PRGM $DEBUGALL $VALID_SEARCH_ENGINES
+    $DEFAULT_SEARCH_ENGINES $RMBLAST_DIR $RMBLASTN_PRGM $RMBLASTDB_PRGM );
+
+%EXPORT_TAGS = ( all => [ @EXPORT_OK ] );
+@ISA         = qw(Exporter);
+
+BEGIN {
+##----------------------------------------------------------------------##
+##     CONFIGURE THE FOLLOWING PARAMETERS FOR YOUR INSTALLATION         ##
+##                                                                      ##
+##
+## RepeatMasker Location
+## ======================
+## The path to the RepeatMasker programs and support files
+## This is the directory with this file as well as
+## the ProcessRepeats and Library/ and Matrices/ subdirectories
+## reside.
+##
+##    i.e. Typical UNIX installation
+##     $REPEATMASKER_DIR = "/usr/local/RepeatMasker";
+##    Windows w/Cygwin example:
+##     $REPEATMASKER_DIR = "/cygdrive/c/RepeatMasker";
+##
+  $REPEATMASKER_DIR          = $ENV{'REPEATMASKER_DIR'};
+  $REPEATMASKER_MATRICES_DIR = "$REPEATMASKER_DIR/Matrices";
+  $REPEATMASKER_LIB_DIR      = "$REPEATMASKER_DIR/Libraries";
+
+##
+## Search Engine Configuration:
+##   RepeatMasker uses either the CrossMatch, WUBlast/ABBlast, or the
+##   TimeLogic search engine to find matches to interspersed
+##   repeat consensi.  You are only required to have one engine
+##   installed on your system in order to run RepeatMasker.  
+##   
+##   The optional program RepeatProteinMask will only run
+##   with the WUBlast/ABBlast package ( currently ).  
+##
+
+  ##
+  ## CrossMatch Location
+  ## ===================
+  ## The path to Phil Green's cross_match program ( phrap program suite ).
+  ##   - Use cross_match version 980501 or later for best results
+  ##   - On a windows machine running the cygwin emulation software
+  ##     you might set this to something like this:
+  ##
+  ##            $CROSSMATCH_DIR = "/cygdrive/c/phrap";
+  ##            $CROSSMATCH_PRGM = "cross_match.exe";
+  ##
+    $CROSSMATCH_DIR = "/usr/local/bin";
+    $CROSSMATCH_PRGM = "$CROSSMATCH_DIR/cross_match";
+
+  ##
+  ## HMMER Location
+  ## ========================
+  ## Set the location of the HMMER programs and support utilities.
+  ##
+    $HMMER_DIR   = $ENV{'HMMER_DIR'};
+    $NHMMSCAN_PRGM = "$HMMER_DIR/nhmmscan";
+    $HMMPRESS_PRGM = "$HMMER_DIR/hmmpress";
+
+  ##
+  ## RMBlast Location
+  ## ========================
+  ## Set the location of the NCBI RMBLAST programs and support utilities.
+  ##
+    $RMBLAST_DIR   = $ENV{'RMBLAST_DIR'};
+    $RMBLASTN_PRGM = "$RMBLAST_DIR/rmblastn";
+    $RMBLASTX_PRGM = "$RMBLAST_DIR/blastx";
+    $RMBLASTDB_PRGM   = "$RMBLAST_DIR/makeblastdb";
+ 
+  ##
+  ## WUBLAST/ABBlast Location
+  ## ========================
+  ## Set the location of the WUBLAST/ABBlast programs and support utilities.
+  ##
+    $WUBLAST_DIR   = "/usr/local/abblast";
```

Comment:
Those paths are not used since rmblast is used as the search engine, not wublast or crossmatch.